### ISSUE TITLE
Completion doesn't show non-static local method in static method

### ISSUE THREAD

**Issue by huoyaoyuan** [dotnet/roslyn#39259]
**Version Used**: 
Visual Studio 16.3.4

**Steps to Reproduce**:

```csharp
class Program
{
    static void Main(string[] args)
    {
        Console.WriteLine("Hello World!");
        int X(int y) => y + 1;
        static int XS(int y) => y + 1;
        // caret here
    }

    void M()
    {
        int X(int y) => y + 1;
        // caret here, for comparison
    }
}
```

**Expected Behavior**:
Both `X` and `XS` should be available in the static method.
**Actual Behavior**:
Only `X` is available.

There should be misusage of `IsStatic` on static local methods.

**Comment by svick** [comment]
This looks like a duplicate of https://github.com/dotnet/roslyn/issues/38074, which should be fixed in VS 16.4 Preview 1.

**Comment by huoyaoyuan** [comment]
Quite sure of duplication, but I cannot see that issue when searching for "completion".
